app/models/course_enrichment.rb

```diff
@@ -27,7 +27,6 @@ def draft?
   end
 
   # About this course
-  # TODO POST MIGRATION: Move out of this as it's handled in the form object
```

Comment:
?

Reply:
I removed these as I realise we need to keep the related validations since they get enforced when a course is attempted to be published.